extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -96,6 +96,36 @@
     @ConfigItem(name = "auth.session.encryption-key")
     public Optional<String> encryptionKey;
 
+    /**
+     * Enable socket reuse port (linux/macOs native transport only)
+     */
+    @ConfigItem(defaultValue = "false", name = "so-reuse-port")
+    public boolean soReusePort;
```

Comment:
Any reason to set the name? The name you use are the default ones.

Reply:
You're right. I'll fix this.